ethpm_cli/commands/install.py

```diff
@@ -119,6 +129,93 @@ def get_package_aliases(package_name: str, config: Config) -> Iterable[Tuple[str
                 yield alias
 
 
+def resolve_pkg_name_or_alias(pkg_name: str, config: Config) -> InstalledPackage:
```

Comment:
I'd suggest two things.

1. Don't abbreviate and use `_package_` and stick with that through all of the naming you do as often as possible.  It will alleviate confusion and mental overhead if you just stick with the long form and thus, never have to remember which ones use the abbreviated version and which don't.
2. Maybe name it `resolve_installed_package` which I think better communicates what this is doing.  On my first read I had to dig into the function body to figure it out as it wasn't clear what resolve meant.  (this also implies renaming the argument `pkg_name -> package_name_or_alias` or maybe `package_identifier` if you want to use that naming convention to represent *either*.